src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/NewAzureRmDataLakeAnalyticsAccount.cs

```diff
@@ -54,6 +54,24 @@ public class NewAzureDataLakeAnalyticsAccount : DataLakeAnalyticsCmdletBase
         [ValidateNotNull]
         public Hashtable Tags { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 5, Mandatory = false,
```

Comment:
@begoldsm more than four positional parameters for a cmdlet is too many. Please get ready of the position for the parameters you are adding to this cmdlet